modules/tensor_mechanics/src/actions/GeneralizedPlaneStrainAction.C

```diff
@@ -41,12 +41,10 @@ GeneralizedPlaneStrainAction::act()
 {
   if (_current_task == "add_kernel")
   {
-    std::string k_type("GeneralizedPlaneStrainOffDiag");
+    std::string k_type = "GeneralizedPlaneStrainOffDiag";
     InputParameters params = _factory.getValidParams(k_type);
-
     params.set<std::vector<NonlinearVariableName> >("displacements") = _displacements;
-
-    params.set<std::vector<VariableName> >("scalar_strain_zz") = {_scalar_strain_zz};
+    params.set<std::vector<VariableName> >("scalar_strain_zz") = { _scalar_strain_zz };
```

Comment:
Is this the MOOSE style for spacing around the curly braces in initializer lists?  Seems inconsistent with the way we don't use spacing around other types of brackets. If this is the preferred way to do this, we need to update the CodeStandards page accordingly because it doesn't say anything about this.